worker/src/data.ts

```diff
@@ -74,5 +93,24 @@ export const createQueueData = (): QueueData => ({
   count_users: [],
 });
 
+export const generateMetadata = (
+  payload: SanitizedPayload,
```

Comment:
SanitizedPayload already has already limited installation_type to valid versions and made sure it's a valid version and also made sure it's capped at X chars ?

Reply:
yes

Reply:
What about the HA version number?

Reply:
And country is passed in from CF right, so a user can't mess with it.

Reply:
CF provides the country (and eventually state), I guess we should validate the version.